projects/plugins/jetpack/class-jetpack-recommendations-banner.php

```diff
@@ -274,4 +276,18 @@ private function render_checkbox( $name, $title ) {
 		</label>
 		<?php
 	}
+
+	/**
+	 * Returns the path of the banner image for the specified version.
+	 *
+	 * @param string $res Requested resolution.
+	 * @param string $format Requested format.
+	 * @return string Path
+	 */
+	private function img_path( $res = 1, $format = 'png' ) {
+		$suffix = 2 === $res ? '-2x' : '';
+		$ext    = in_array( $format, array( 'webp', 'png' ), true ) ? $format : '';
```

Comment:
Should there be a default extension instead of an empty string, so the path remains valid?